docs/use_cases/model_applications/marine_and_cryosphere/GridStat_fcstRTOFS_obsAVISO_climHYCOM_ssh.py

```diff
@@ -86,7 +86,7 @@
 
 ##############################################################################
 # MET Configuration
```

Comment:
@lisagoodrich In this section, please remove line 101 (.. highlight:: bash) as the template does not include that line.